script/job_generator/generate_job_yaml.py

```diff
@@ -391,6 +391,10 @@ def job_variables(job: Dict[str, Tuple[str, str]]) -> Dict[str, str]:
         variables["ALPAKA_CI_ONEAPI_VERSION"] = job[DEVICE_COMPILER][VERSION]
         variables["alpaka_SYCL_ONEAPI_CPU"] = "ON"
         variables["alpaka_SYCL_ONEAPI_CPU_ISA"] = "avx2"
+        variables["alpaka_SYCL_ONEAPI_FPGA"] = "ON"
```

Comment:
Can you please add a `TODO` that this should be split in a separate Intel SYCL CPU and FPGA job in future with the next commit.